Proposal for "information_uri" (#3)
peppelinux/openid-federation-subordinate-events: openid-federation-subordinate-events-1_0.md

- **iat**: (REQUIRED) Time when the event is related to, using the time format defined for the `iat` claim.
- **event**: (REQUIRED) String that identifies the event, such as `registration`, `jwks_update`, `metadata_policy_update`, `metadata_update`, or `revocation`.
- **event_description**: (OPTIONAL) String that may offer additional information about the event.

> - **information_uri**: (OPTIONAL) URI that may offer additional information about the event.

Reviewer: we should say that it SHOULD be an https url, do we agree?

Author: Yes

> - **information_uri**: (OPTIONAL) URI that may offer additional information about the event.

Reviewer: ```suggestion
- **information_uri**: (OPTIONAL) URI that may offer additional information about the event. Use of HTTPs URLs is RECOMMENDED.
```

> - **information_uri**: (OPTIONAL) URI that may offer additional, human readable, information about the event. Use of HTTPs URLs is RECOMMENDED.

Reviewer 2: ```suggestion
- **information_uri**: (OPTIONAL) URL for documentation of additional information about this event viewable by the End-User. Use of HTTPs URLs is RECOMMENDED.
```

Use same wording as `information_uri` at [Informational Metadata Extensions](https://openid.net/specs/openid-federation-1_0.html#name-informational-metadata-exte).

Author: Agreed, though I would suggest to replace "Entity" by "Event"

Reviewer 2: Done

  "federation_registration_events": [
    {
      "iat": 1590000000,
      "event": "registration",

>       "information_uri" : "https://immediate-superior.example.org/policy/Metadata+Registration+Practice+Statement"

Reviewer 2: ```suggestion
      "information_uri" : "https://immediate-superior.example.org/policy/Metadata_Registration_Practice_Statement"
```

Let's change the plusses to underscores, since + is used in form-urlencoding for spaces.
